core/src/main/java/com/oath/oak/Slice.java

```diff
@@ -12,7 +12,7 @@
 
 // Represents a portion of a bigger block which is part of the underlying managed memory.
 // It is allocated via block memory allocator, and can be de-allocated later
-class Slice implements Comparable<Slice> {
+class Slice implements OakUnsafeDirectBuffer, Comparable<Slice> {
```

Comment:
Wasn't it already? Weird...

Reply:
Before the new-api, the implementation of `OakUnsafeDirectBuffer.getByteBuffer()` would have been oakward (pun intended) in `Slice`, so we couldn't have supported the full `OakUnsafeDirectBuffer` in `Slice`.
Now it is straight forward.